app/views/users/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<a href="<%= user_posts_path(@user) %>">
```

Comment:
- [ ] Here as well please instantiate to @user variable in the controller's show method first to use it in the views/show page of the users.

Reply:
This change has been done well